src/vm/gdbjit.cpp

```diff
@@ -1116,6 +1150,109 @@ void FunctionMember::DumpStrings(char* ptr, int& offset)
     DumpLinkageName(ptr, offset);
 }
 
+bool FunctionMember::GetBlockInNativeCode(int blockILOffset, int blockILLen, TADDR *startOffset, TADDR *endOffset)
+{
+    PCODE pCode = md->GetNativeCode();
+
+    const int blockILEnd = blockILOffset + blockILLen;
+
+    *startOffset = 0;
+    *endOffset = 0;
+
+    bool in_block = false;
```

Comment:
Could you please use locals naming convention consistent with the surrounding stuff at all places you have changed?